scripts/cmake/addons.cmake

```diff
@@ -0,0 +1,63 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### Helper function to build a VPN Addon
```

Comment:
Does this mean we can use cmake target=addon to generate them? And get rid of the scripts?

Reply:
Yeah, that is one of my goals in this PR before I'll consider it ready for review.